src/hooks/http.hook.js

```diff
@@ -0,0 +1,36 @@
+export const useFetch = () => {
+    const request = async (url, method = 'GET', body = null, headers = {'Content-Type': 'application/json'}) => {
```

Comment:
Unfortunaltely, it's not a hook. It's just a simple utility function. Hook should contain usage of other react or custom hooks